lib/models/addressinfo.js

```diff
@@ -27,6 +27,9 @@ function AddressInfo(param) {
 }
 
 AddressInfo.fromInsight = function(param) {
+  if (_.isString(param)) {
```

Comment:
Instead of JSON parsing here where an error in parsing would cause a crash, should we try/catch at: https://github.com/bitpay/bitcore-explorers/blob/master/lib/insight.js#L122 and then pass an error in the callback?